src/style.css

```diff
@@ -1,3 +1,47 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.logo__img {
+  display: inline-block;
+  position: absolute;
+  margin: 10px 50px;
+}
+
+.nav__list {
+  display: flex;
+  position: absolute;
```

Comment:
Using `position: absolute;` for the navigation list can lead to layout issues, especially on responsive designs. Consider using a more flexible layout approach, such as flexbox or grid, to ensure the navigation is positioned correctly across different screen sizes.